Separa guía de autores y traductores + agrega lineamientos sobre corrección de estilo y formato

mariajoafana opened the pull request:
*Estoy creando un archivo de trabajo para traducir y adaptar la nueva versión de la guía para autores del equipo en español. El nombre del archivo es "guia-para-autores-en-proceso" y lo trabajaremos en jekyll/es/ . Cuando esté listo reemplazaremos en actual archivo "guia-para-autores"*

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Assign at least one individual or team to "Reviewers"
- [x] if the text needs to be translated, assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing editor in your PR. Please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines) when writing your PR description
- [x] Add the appropriate "Label"
- [x] [Ensure the Travis CI checks pass](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#4-check-your-pr-status)
- [x] [Check the live preview of your PR on Netlify](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#5-preview-how-your-pr-looks-when-built-into-html)
- [ ] If this PR closes an open issue, add the phrase `Closes #ISSUENUMBER` to the description above

*If you are having difficulty fixing Travis errors, first consult <https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions> carefully, especially ["Common Travis Errors"](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#common-travis-errors). Then contact the technical team if you need further help.*

mariajoafana committed:
Create guia-para-autores-en-proceso

mariajoafana committed:
Update guia-para-autores-en-proceso

rivaquiroga commented:
Buenísimo! @mariajoafana,  ¿te parece si dejamos el PR como draft por ahora?

mariajoafana commented:
Si @rivaquiroga dejémoslo como draft. Espero de acá al fin de semana hacer la primera ronda de toda la traducción/adaptación y avisarte para que lo revises, ¿te parece?

mariajoafana committed:
Update guia-para-autores-en-proceso

rivaquiroga commented on es/guia-para-autores-en-proceso:
```diff
@@ -56,38 +56,38 @@ Esta primera sección se ocupa de cuestiones de estilo generales que te ayudará
-### Dates and Time
- *	For centuries, use eighteenth century not 18th century. Avoid national-centric phrases such as "long eighteenth century" which have specific meaning to British eighteenth century specialists, but not to anyone else.
- *	For decades, write the 1950s (not "the 1950s" or "the fifties").
+### Fechas y hora
```
@mariajoafana, acá sugeriría que dejes por ahora solo la lista con los temas, es decir:
* siglos
* decádas
* etc.
Las convenciones en español son distintas del inglés en esto, así que no vale la pena traducirlas. (Por ejemplo, los siglos en español se escriben con números romanos)

rivaquiroga replied:
El uso de mayúsculas también es distinto, así como algunas convenciones para escribir números y abreviaciones

mariajoafana committed:
Update guia-para-autores-en-proceso

mariajoafana commented on es/guia-para-autores-en-proceso:
```diff
@@ -0,0 +1,344 @@
+## B. Pautas específicas de escritura
+En esta segunda sección se tratan cuestiones más específicas del estilo de escritura, como qué palabras utilizar, o cómo usar la puntuación, o qué formato utilizar para las fechas o los números. Lea esta sección antes y después de escribir su borrador.
+
+### Fechas y hora
```
La RAE desaconseja utilizar el orden de fecha habitual para los países anglosajones. Solamente lo aconseja para documentos de carácter científico o técnico de circulación internacional. Acá tendremos que tomar la decisión entonces: seguimos las normas ISO, o a la RAE para el formato de fechas. Acá lo que dice la RAE: https://www.rae.es/dpd/fecha

mariajoafana committed:
Update guia-para-autores-en-proceso

mariajoafana committed:
Update guia-para-autores-en-proceso

mariajoafana committed:
Update guia-para-autores-en-proceso

mariajoafana commented:
Bueno, acabo de terminar una primera ronda de traducción pero falta aun trabajo para ajustar esto a las particularidades del español. En varias sección añadí [REVISAR] e incluí algunos enlaces de la RAE para ayudarnos a incorporar las reglas que aplican al español. 
Hay secciones que comencé pero que necesitan más trabajo para terminarlas. Ahí quisiera pedirles el apoyo a quienes tengan el tiempo y disposición para sacar esto adelante. Las enumero a continuación: 

1. Sección B: 
- Fechas y horas 
- Números
- Abreviaturas 
- Mayúsculas 

2. Revisar ejemplos propuestos para toda la guía que tengan sentido para la audiencia hispanohablante (añadí algunos pero habría que revisar y proponer nuevas cosas si es necesario)

3. Pensar en la sección "Explicación de palabras difíciles", ¿qué tiene sentido para el español acá?

4. Trabajar en la sección "Sigue las mejores prácticas para escribir tu código (línea 272). Acá sugiero que alguien con más experiencia con código lo revise.

5. Revisar para ver ¿qué normas gramaticales esenciales están faltando en la guía de estilo?

6. Corrección de estilo (creo que no fui todo el tiempo consistente con el "tu" y el "usted", esta es una de las cosas para revisar). 

Por el momento esto es lo que se me ocurre.

rivaquiroga commented:
Muchas gracias @mariajoafana! Sobre tu/usted, sería bueno también tomar una decisión al respecto en términos de estilo de los tutoriales. En la mayoría (¿o en todos?) se tutea, así que quizás deberíamos incluir también eso en la guía. 
Mañana puedo darle una mirada y avanzar con las secciones pendientes.
Nuevamente, muchas gracias!

rivaquiroga committed:
revisión parcial + traducción de otras secciones

rivaquiroga commented:
@mariajoafana, revise los puntos 4 y 6. Estoy buscando ejemplos y lineamientos para los otros. 

En este PR también deberíamos resolver este _issue_: https://github.com/programminghistorian/jekyll/issues/1695

El nombre de la rama (`mariajoafana-patch-1`) no es muy descriptivo. Si te parece, podrías cambiarlo a algo como `guia-estilo-es`

mariajoafana commented:
> @mariajoafana, revise los puntos 4 y 6. Estoy buscando ejemplos y lineamientos para los otros.
> 
> En este PR también deberíamos resolver este _issue_: #1695
> 
> El nombre de la rama (`mariajoafana-patch-1`) no es muy descriptivo. Si te parece, podrías cambiarlo a algo como `guia-estilo-es`

Sii, de acuerdo, no me acordé de crear un branch específico, pero ya no se cómo cambiarle el nombre.

mariajoafana commented:
Acabo de traducir la guía para traducciones acá: #1806 
@jamotilla y @rivaquiroga  ¿Pueden hacerle una revisión para aprobar su publicación?

jamotilla commented:
Hola @mariajoafana , revisé la traducción y tengo las siguientes observaciones:

Línea 18:  “incluyen detalles prácticos y filosóficos” / no se si “filosófico” sea el término adecuado
29: “completa una el” / suprimir “una”
67: “Tén” / "Ten" 
70: “elije” / elige
71: “Haz” / has
75: Pensando “en largo plazo” / “pensando a largo plazo”
91: “comprima” / “comprime”
92: “which conforms to the standard ISO 8601:2004. This avoids ambiguity.” / Falta traducción
97-103: falta traducción 
113-123: revisar
127-176: Como el texto lo consigna, es necesario revisar los aspectos de puntuación y mayúsculas de la RAE
300: “su” / “tu” 
300: “suba” / “sube”
302: “su” / “tu”
313: “su” / “tu”
317: “lrecomendará” / “recomendará

Respecto a las pautas de la RAE, si te parece yo me encargo de buscarlas las agrego en los comentarios para que puedas agregarlos ¿te parece?

mariajoafana committed:
Update guia-para-autores-en-proceso

Acabo de incorporar las correcciones 
@jamotilla quedan faltando las pautas de la RAE para 127-176 aspectos de puntuación y mayúsculas de la RAE

mariajoafana committed:
Update guia-para-autores-en-proceso

rivaquiroga committed:
revisión + sección sobre lenguaje inclusivo

rivaquiroga committed:
Delete guia-para-autores-en-proceso

El primer archivo sobre el que trabajamos estaba sin extensión. Al subir los cambios desde mi computador quedó duplicado, así que dejé la versión.md

rivaquiroga committed:
agrega punto 5 en las indicaciones sobre el envío

Los otros cuatro ya los habíamos incorporado cuando empezamos a elaborar esta actualización

jenniferisasi commented:
Por el comentario de @rivaquiroga en #1844 creo que este documento está finalizado? Si es así, por favor haced el Merge para luego reflejar/añadir la frase extra de #1844. Me decís si tenéis problemas con Travis

rivaquiroga commented:
Mañana jueves hago el merge.

rivaquiroga committed:
separa guia autores y traductores

rivaquiroga committed:
actualiza plantilla lecciones

rivaquiroga committed:
remueve url

rivaquiroga committed:
fix typo

rivaquiroga committed:
más ediciones en la plantilla

rivaquiroga committed:
fix link

rivaquiroga commented:
This PR is ready to merge after @programminghistorian/technical-team helps us with the spliting: https://github.com/programminghistorian/jekyll/issues/1695#issuecomment-593573275

walshbr committed:
Merge branch 'gh-pages' into mariajoafana-patch-1

walshbr committed:
Account for split of spanish docs

walshbr commented:
I think I took care of it, but you should double check. I had to add a snippet for the title of the new translator page. So please check my Spanish - https://github.com/programminghistorian/jekyll/pull/1771/commits/b70c251c14d1bc31dd53bd0b5e9d8f2376adc3d7. One other thing is that this is what the menu looks like now - 
![Screen Shot 2021-01-11 at 10 10 48 AM](https://user-images.githubusercontent.com/2334220/104199481-8053ca80-53f5-11eb-9fc7-193e68443516.png).

You probably want to change the title of the original page, right? It still reads "Guía para autores y traductores" when it should probably just be "Guía para autores." If so, that change needs to happen in two places - line 2 of /es/guia-para-autores.md and line 158 of _data/snippets.yml. Let me know if you need more help.

rivaquiroga committed:
update title of Guía para autores

rivaquiroga committed:
actualiza título Guía para autores

rivaquiroga commented:
Thanks, @walshbr!

rivaquiroga committed:
fix typos en la guía para autores

rivaquiroga committed:
fix typos en la guía para traductores
